src/04-modality-specific-files/29-motion.md

```diff
@@ -0,0 +1,224 @@
+
+The columns of the channels description table stored in `*_channels.tsv` are:
+
+MUST be present:
+
+| **Key name**  | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                |
+| ------------- | --------------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| component     | REQUIRED              | string        | Component of the representational system described in `*_coordinatesystem.tsv` that the channel contains.                                                                                      |
+| name          | REQUIRED              | string        | Label of the channel. Entries have to match headers in (any) `*_motion.tsv.`                                                                                                                   |
+| tracked_point | REQUIRED              | string        | Label of the point that is being tracked, for example, label of a tracker or a marker (`LeftFoot`, `RightWrist`).                                                                              |
+| type          | REQUIRED              | string        | Type of data.                                                                                                                                                                                  |
+| units         | REQUIRED              | string        | Physical unit of the value represented in this channel, for example, vm for virtual meters, radian or degrees for angular quantities. See the BIDS spec for guidelines for Units and Prefixes. |
+
+SHOULD be present:
+
+| **Key name**       | **Requirement level** | **Data type** | **Description**                                                                                                                                                             |
+| ------------------ | --------------------- | ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| placement          | RECOMMENDED           | string        | Placement of the tracked point on the body (for example, participant, avatar centroid, torso, left arm). It can refer to an external vocabulary for describing body parts.  |
```

Comment:
Similar to the comment above, would this be related to the BodyPart, BodyPartDetails, BodyPartDetailsOntology in the Microscopy specs?

Reply:
For motion capture data there is currently no standard that labs can agree on. I have been working on a preprint to tackle this problem, but it is a slow work in progress :D. We will check if we can still use the BodyPart keys in our BEP.